### DISCUSSION TITLE
How do I make this app work? (Newbie)

### DISCUSSION THREAD

**Discussion by ItsRadysdxas** [pear-devs/pear-desktop#111] [Q&A] [answered]
I don't know anything about programming so I don't understand what is being told here. I downloaded the zip file but I don't know how to run it or make it work. I downloaded the file because I was interested in the Adblock function that it provides. I looked in ReadMe but still don't know what to do.

Edit: It works now :)

**Comment by danielle-segal** [comment]
Hi, can u help me with the Adblock also? I can't understand the readme file at all.

**Reply by ItsRadysdxas** [reply]
Hi! You have to go to "latest release"(latest release is at this link if you can find it: https://github.com/th-ch/youtube-music/releases/tag/v1.8.0) and click on "YouTube-Music-Setup-1.8.0.exe" to download it. You'll get a setup file which you open and follow the instructions after that it's installed. If the Adblock plugin doesn't work try to change the location to another country.

**Reply by th-ch** [reply]
Hey @danielle-segal, thanks for the question! For the adblocker, also ensure that it is enabled (you can check in the `Plugins` menu) and from there, it should work out of the box - Feel free to post here if something does not work as expected!

**Comment by th-ch** [comment, marked as answer]
> Edit: It works now :)

Marking this question as answered, but feel free to post if something does not work as expected!
